src/resolvers/projectResolver.ts

```diff
@@ -538,11 +538,11 @@ export class ProjectResolver {
       });
     query = ProjectResolver.addUserReaction(query, connectedWalletUserId, user);
     const project = await query.getOne();
-    const userId = connectedWalletUserId || user?.userId;
 
     if (
       project?.statusId === ProjStatus.drafted &&
-      (!userId || project?.admin !== String(userId))
```

Comment:
@CarlosQ96 
You should have checked logged in userId not `connectedWalletUserId` because use can send `connectedWalletUserId` and view draft projects of other users